examples/low-level-api/create_vc.rs

```diff
@@ -41,11 +44,12 @@ pub async fn create_vc() -> Result<()> {
   println!("Credential JSON > {:#}", credential);
 
   // Validate the verifiable credential
-  let validation: CredentialValidation = common::check_credential(&client, &credential).await?;
-  println!("Credential Validation > {:#?}", validation);
-  assert!(validation.verified);
-
-  Ok(())
+  //Todo: Use the new resolver to get the necessary DID documents once that becomes available.
+  let validator = CredentialValidator::new();
+  let validation_options = CredentialValidationOptions::default();
+  let resolved_issuer: ResolvedIotaDocument = client.resolve(issuer_doc.id()).await?;
+  let fail_fast = true;
+  validator.full_validation(&credential, &validation_options, &resolved_issuer, fail_fast)
```

Comment:
Yeah, it puts some small additional burden on the user. I think readability trumps writeability, because code is read more often than written, so that seem okay. I'm leaning towards trying it (I've never actually used it, but always wanted to), but I'm fine either way.